cove_bods/test_functional.py

```diff
@@ -30,7 +30,7 @@ def server_url(request, live_server):
 
 
 @pytest.mark.parametrize(('link_text', 'expected_text', 'css_selector', 'url'), [
-    ('Open Ownership', 'Anonymous shell companies', '.home', 'https://openownership.org/'),
+    ('Open Ownership', 'Driving the global shift towards transparency', '.home', 'https://openownership.org/'),
```

Comment:
Why is this test testing the content and DOM structure of the OO homepage? (And presumably relying on the actual live site to do so). Wouldn't just checking that there's a link with the correct URL be sufficient?

Reply:
I think the point is to check it's not a broken link, but I agree this is very unlikely for the OO core domain. Happy to remove if you want.

Reply:
If that's ok, yes please - I think it would be preferable if the tests could run without relying on something external.